src/containers/PipelineRun/PipelineRun.js

```diff
@@ -97,6 +97,21 @@ export /* istanbul ignore next */ class PipelineRunContainer extends Component {
 
   render() {
     const { error, intl, match, pipelineRun, tasks, taskRuns } = this.props;
+
+    if (!this.props.pipelineRun) {
```

Comment:
no need for `this.props`, we've already deconstructed the props to extract `pipelineRun` so you can reference it directly here and below

Reply:
missed this one? few of these:

```diff
-this.props.pipelineRun
+pipelineRun
```